artifactory/commands/packagemanagerlogin/packagemanagerlogin.go

```diff
@@ -239,3 +281,36 @@ func (pmlc *PackageManagerLoginCommand) configureDotnetNuget() error {
 	// Add the repository as a source in the NuGet configuration with credentials for authentication.
 	return dotnet.AddSourceToNugetConfig(toolchainType, sourceUrl, user, password)
 }
+
+// configureContainer configures container managers like Docker or Podman to authenticate with JFrog Artifactory.
+// It performs a login using the container manager's CLI command.
+//
+// For Docker:
+//
+//	docker login <artifactory-url-without-scheme> -u <username> -p <password>
+//
+// For Podman:
+//
+//	podman login <artifactory-url-without-scheme> -u <username> -p <password>
+func (pmlc *PackageManagerLoginCommand) configureContainer() error {
+	var containerManagerType container.ContainerManagerType
+	switch pmlc.packageManager {
+	case project.Docker:
+		containerManagerType = container.DockerClient
+	case project.Podman:
+		containerManagerType = container.Podman
+	default:
+		return errorutils.CheckErrorf("unsupported container manager: %s", pmlc.packageManager)
+	}
+	// Parse the URL to remove the scheme (https:// or http://)
+	parsedURL, err := url.Parse(pmlc.serverDetails.GetUrl())
```

Comment:
Emphasize that it is the platform URL (Am I wrong?)
```suggestion
	parsedPlatformURL, err := url.Parse(pmlc.serverDetails.GetUrl())
```